### PR TITLE
Fix header hugo

### PR DESCRIPTION
**Pull request by moxiegirl**
Same fix on hugo

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @SvenDowideit

File: content/apidocs.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-title = "Generated API documentation"
-description = "Generated API documentation"
-keywords = ["docker, documentation, about, technology, understanding, api, release"]
-+++
-
-# Generated API documentation
-
-There is generated API documentation for:
-
-* [Docker Trusted Registry](apidocs/docker-trusted-registry/index.md)
```
Comment (SvenDowideit) [inline content/apidocs.md]: mmm, this page is being over-written by dtr, so ok, i guess

File: layouts/404.html [hunk 2]
```diff
@@ -1,147 +1,4 @@
...
-          <li><a href="#">Sign in</a></li>
-          <li><a href="#">Sign up</a></li>
-        </ul>
-      </aside>
-      <!-- close the off-canvas menu -->
-      <a class="exit-off-canvas"></a>
-
-
-
+{{ partial "header.html" . }}
```
Comment (SvenDowideit) [inline layouts/404.html]: this won't work, as the header.html uses links that hugo makes relative, which then break when used on s3 with non-root urls

File: layouts/404.html [hunk 3]
```diff
@@ -149,11 +6,10 @@
 	  <div>
           <h1 style="padding-top:15px;" id="title">{{ .Title }}</h1>
           <h2 style="color:#3A4E55;">The page you wanted cannot be found or is no longer available.</h2>
-		  <p>Can we help you find what you're looking for? Use the <a href="{{ .Site.BaseURL }}index.html">sitemap to navigate</a> directly or send email to <a href="mailto:docs@docker.com">docs@docker.com</a> for additional help.</p>
+		  <p>Can we help you find what you're looking for? Use the <a href="index.html">sitemap to navigate</a> directly or send email to <a href="mailto:docs@docker.com">docs@docker.com</a> for additional help.</p>
```
Comment (SvenDowideit) [inline layouts/404.html]: this won't work, as the header.html uses links that hugo makes relative, which then break when used on s3 with non-root urls

File: static/css/custom.css [hunk 4]
```diff
@@ -62,6 +62,25 @@ div#buildinfo {
...
+  margin-bottom: 280px;
+}
+
+.off-canvas-wrap {
+  min-height: 100%;
+}
+
+.inner-wrap {
+  position: static;
+}
```
Comment (SvenDowideit) [inline static/css/custom.css]: no idea what this is intended to fix, so pass :)

File: layouts/404.html (hunk 2 above)
Comment (moxiegirl) [inline layouts/404.html]: As I stated before, this same include is on all our files including the index.html.  It is working there.

Comment (moxiegirl) [inline layouts/404.html]: http://docs-stage.docker.com/index.html

File: content/apidocs.md (hunk 1 above)
Comment (moxiegirl) [inline content/apidocs.md]: I"m going to redo this PR...the rebase on this was borked.

File: static/css/custom.css (hunk 4 above)
Comment (dnephin) [inline static/css/custom.css]: This is taken from my PR #164

I've included a link which shows the problem

**Pull request closed without merging**